inst/staticexports/stringr.R

```diff
@@ -1,13 +1,17 @@
-str_replace <- function(text, pattern, replacement) {
-  sub(pattern = pattern, replacement = replacement, x = text, perl = TRUE)
+str_replace <- function(text, pattern, replacement, fixed = FALSE, perl = !fixed, ...) {
```

Comment:
These new args aren't in `stringr::str_replace`. Why should we add them here?

(Personally, I'd just use `sub()` and `gsub()` instead of these functions.)